packages/create-next-app/templates/default/styles/globals.css

```
@@ -14,3 +14,10 @@ a {
 * {
   box-sizing: border-box;
 }
+
+@media (prefers-color-scheme: dark) {
+  body {
+    color: white;
+    background: black;
+  }
+}
```

```suggestion
@media (prefers-color-scheme: dark) {
  html {
    color-scheme: dark;
  }
  body {
    color: white;
    background: black;
  }
}
```

Toggles scrollbar color to reflect darkmode preferences:

https://user-images.githubusercontent.com/825855/179553119-dab7f4be-5bdf-43ba-8320-6d9c6c4f185c.mp4

MDN color-scheme: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme

---

Lovely! Thank you @alexcarpenter